src/services/axios/index.ts

```diff
@@ -28,7 +28,8 @@ axios.interceptors.response.use(
   (response) => response,
   (error) => {
     if (error.response && error.response.status === 401) {
-      const redirect = encodeURIComponent(location.href);
+      const href = location.href.match('/error') ? location.origin : location.href;
```

Comment:
Does location origin allways exist?

Reply:
Yes, it's a standard property on the location instance.
 https://developer.mozilla.org/en-US/docs/Web/API/Location/origin

Reply:
Excellent, thanks!